app/businesses/models.py

```diff
@@ -27,41 +27,63 @@ def __repr__(self):
         return '<Category {}>'.format(self.name)
 
 
+tags = db.Table('tags',
+                db.Column('tag_id', db.Integer, db.ForeignKey('tag.id'), primary_key=True),
+                db.Column('business_id', db.Integer, db.ForeignKey('business.id'), primary_key=True)
+                )
+
+
 class Business(db.Model, TimestampMixin):
     """
     Business model
     """
-    __tablename__ = 'business'
+    class StatusEnum(Enum):
+        pending = "pending"
+        accepted = "accepted"
+        refused = "refused"
 
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String(), unique=True, nullable=False)
-    legal_name = db.Column(db.String(), unique=True, nullable=True)
-    phone = db.Column(db.String())
     description = db.Column(db.String())
     slogan = db.Column(db.String())
     website = db.Column(db.String(), nullable=True)
     email = db.Column(db.String(), nullable=True)
-    accepted = db.Column(db.Boolean, default=False)
+    status = db.Column(ChoiceType(StatusEnum, impl=db.String()),default="pending")
+    accepted_at = db.Column(db.DateTime, nullable=True, default=None)
     notes = db.Column(db.String(), nullable=True)
     capacity = db.Column(db.Integer, nullable=True)
 
     business_hours = db.relationship('BusinessHour', backref='business', lazy=True)
-    address = db.relationship('Address', backref='business', lazy=True, uselist=False)
-    phones = db.relationship('Phone', backref='business', lazy=True, uselist=False)
-
+    address = db.relationship('Address', backref='business', lazy=True)
+    phones = db.relationship('Phone', backref='business', lazy=True)
+    social_links = db.relationship('SocialLink', backref='business', lazy=True)
+    tags = db.relationship('Tag', secondary=tags, lazy='subquery',
+                           backref=db.backref('pages', lazy=True))
     owner_id = db.Column(db.Integer, db.ForeignKey("owner.id"), nullable=True)
     category_id = db.Column(db.Integer, db.ForeignKey("category.id"), nullable=False)
     payment_type = db.Column(db.Integer, db.ForeignKey("payment_type.id"), nullable=True)
```

Comment:
`payment_type` devrait aussi etre un champs enum. On les connaient d'avance

Reply:
en faites, je repense a ca. On a une relations Many to Many. donc vais corriger